### ISSUE TITLE
Relay temporary not avalaible

### ISSUE THREAD

**Issue by pdifeo** [rustdesk/rustdesk-server-pro#102]
I configured RustDesk with two relays. One himself and another.
For maintenance reasons the additional relay server is down. I cannot connect to all peers. It's normal ?

**Comment by rustdesk** [comment]
How about?

![image](https://github.com/rustdesk/rustdesk-server-pro/assets/71636191/fb7bce59-5800-4934-b3ad-0c55ed3447fa)

**Comment by pdifeo** [comment]
Return the server that in this moment is down

**Comment by pdifeo** [comment]
![image](https://github.com/rustdesk/rustdesk-server-pro/assets/8618744/fb16b260-871e-4203-9f2a-2f72e7c99eed)

**Comment by pdifeo** [comment]
![image](https://github.com/rustdesk/rustdesk-server-pro/assets/8618744/c5c981fb-2896-4074-8929-951a1f2eadaa)

**Comment by pdifeo** [comment]
More than an hour has passed. Otherwise I would not have posed the problem.

**Comment by rustdesk** [comment]
I just tested, it worked as expected. Could you confirm `telnet <the_ip_which_is_down> 21117`? Our health check just telnet the port.

**Comment by rustdesk** [comment]
@Mouxy do you have similar issue?

**Comment by rustdesk** [comment]
@pdifeo could you try using ip rather than domain?

**Comment by pdifeo** [comment]
```
telnet rd-relayxxxxxxxxxxxxx 21117
Trying 217.x.x.x ...
telnet: connect to address 217.x.x.x: Connection refused
```

**Comment by pdifeo** [comment]
> @pdifeo could you try using ip rather than domain?

tomorrow, now I'm updating the hosting machine.

**Comment by rustdesk** [comment]
I will test domain names also tomorrow.

**Comment by Mouxy** [comment]
It only work with IP addresses, I mentioned that in the docs

**Comment by pdifeo** [comment]
> It only works with IP addresses, I mentioned that in the docs

Not good and I'll explain why

I use the domain name because some clients are on the same LAN as the relay.

So to make the scenario clear to you, this relay is with private address. Necessarily I had to assign a domain name. On the Internet it is solved with the public IP address. On LAN it is resolved with private ip.

Clearly the server is under NAT 1:1

From what you say, a relay cannot be under NAT and in the same network as some clients.

**Comment by rustdesk** [comment]
@Mouxy Thanks for your clarification.

**Comment by Mouxy** [comment]
![image](https://github.com/rustdesk/rustdesk-server-pro/assets/642149/cc06d838-d4fe-4182-aa71-91647cb6cf3d)

That's it, it should work fine with the IP addresses, I'll make sure to state it doesn't work with names.

I do believe this could be fixed though 😊

**Comment by pdifeo** [comment]
> I do believe this could be fixed though

I very much hope so. I was planning to install a relay at each of my clients or so. If the problem is not solved, I cannot implement my plan.

**Comment by Mouxy** [comment]
Why would you install a relay in every client?

**Comment by pdifeo** [comment]
> Why would you install a relay in every client?

In every site of my customers. Before I wrote wrong. "Cliente" in Italian is Customer.

**Comment by pdifeo** [comment]
I have clients in Italy, Spain and France. So it makes a little sense what I want to do.

But I honestly see inefficiency in choosing a relay based on geographic location. I might get a faster connection by going through another relay because there might be less latency and not the one closest geographically. it is just my opinion which as such is debatable.

**Comment by dinger1986** [comment]
> ![image](https://user-images.githubusercontent.com/642149/272339924-cc06d838-d4fe-4182-aa71-91647cb6cf3d.jpeg)
> 
> That's it, it should work fine with the IP addresses, I'll make sure to state it doesn't work with names.
> 
> I do believe this could be fixed though 😊

Relay servers do work with names, I have setup around 100 RustDesk Server Pro's on VPS both testing and installing for other people, DNS works fine if setup ONLY for ipv4 and when added in the relay box its added without the port, (tbf never tried it with the port).

**Comment by dinger1986** [comment]
further proof it worked (I never touched that server) https://github.com/rustdesk/rustdesk/discussions/5817#discussioncomment-7180781

TLDR:
Not working for OP or relaying sessions, kept on getting failed when trying to connect, added in relay server under web console as dns name and working immediately after

**Comment by rustdesk** [comment]
I think we may solve >1 relay servers with dns in next releases.

**Comment by pdifeo** [comment]
@dinger1986 Perhaps the problem is not clear. If the ADDITIONAL relay server is DOWN, nothing works anymore. It normally works fine.  I keep the problem. So some thing makes it happen.

**Comment by rustdesk** [comment]
This issue is very clear I think, no need to discuss it further which making it more confusing.

**Comment by pdifeo** [comment]
> I think we may solve >1 relay servers with dns in next releases.

Hi 
Any news about it ?

**Comment by pdifeo** [comment]
Hi,

when is implementation planned? is very important to me.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk-server-pro/releases/tag/1.1.11
